src/routes/blog/index.svelte

```diff
@@ -1,9 +1,25 @@
 <script lang="ts" context="module">
   import type { LoadInput, LoadOutput } from '@sveltejs/kit/types/internal';
+  import TagField from '$lib/components/utils/acm-tagfield.svelte';
 
-  export async function load({ fetch }: LoadInput): Promise<LoadOutput> {
-    const response = await fetch(`/blog.json`);
-    return { props: { posts: await response.json() } };
+  let tags = [];
+  let selectedTags: string[] = [];
+
+  export async function load(event: LoadInput): Promise<LoadOutput> {
```

Comment:
Nit: Since we only use `event` to access `event.url`, let's deconstruct `url` out right away.
```suggestion
  export async function load({ url }: LoadInput): Promise<LoadOutput> {
```

Reply:
I fixed it.